dev/shim/dune

```diff
@@ -18,7 +18,7 @@
   (with-stdout-to %{targets}
    (progn
     (echo "#!/usr/bin/env bash\n")
-    (bash "echo '\"$(dirname \"$0\")\"/%{bin:coqtop} -I \"$(dirname \"$0\")/%{workspace_root}/_build/install/default/lib\" -coqlib \"$(dirname \"$0\")/%{project_root}\" \"$@\"'")
+    (bash "echo '\"$(dirname \"$0\")\"/%{bin:coqtop} -I \"$(dirname \"$0\")/%{workspace_root}/../install/default/lib\" -coqlib \"$(dirname \"$0\")/%{project_root}\" \"$@\"'")
```

Comment:
How can this ever make sense? This seems to point to somewhere outside of the workspace.

Reply:
workspace_root is _build/default, we want to point to _build/install/default

Reply:
A OK. What confused me is that I always thought of  `%{workspace_root}` as the actual workspace root (not within the `_build` directory, but since it is expanded to a relative path both interpretations make sense.